docs/pages/Theming/CustomTheme.md

```diff
@@ -1,7 +1,104 @@
+
+### Components
+
+The FURN theme has a components property which can be used to customize components:
+
+```ts
+interface Theme {
+  ...
+  components: {
+    [key: string]: object,
+  };
+  ...
+}
+```
+
+<font size=1>(Taken from the [Theme type definition](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/theme-types/src/Theme.types.ts).)</font>
+
+In order to have the theme override a particular component's styling, you'll need to specify a component's name and provide some overrides for the component's settings. The settings are an object that specifies how a component's tokens or slot's style are filled out.
+
+```ts
+components: {
+  <ComponentName>: {
+    <tokenName>: <overrideValue>
+  },
+},
+```
+
+For example, if you want to change the settings on the [Button component](https://github.com/microsoft/fluentui-react-native/blob/master/packages/components/Button/src/Button.tsx), you could do something like the following:
+
+```ts
+components: {
+  Button: {
+    backgroundColor: 'primaryButtonBackground', // Different semantic color
+    color: 'neutralBackground1', // Alias token
+    hovered: {
+      backgroundColor: theme.host.colors['AppShade10'], // Get brand color from host
+      color: theme.host.colors['Gray96'], // Get gray from host
+      borderColor: theme.host.palette.Bkg, // Get entry from palette from host
+    },
+  },
+},
+```
+
+You can see what settings can be customized by looking at a component's settings file. For Button, that can be found [here](https://github.com/microsoft/fluentui-react-native/blob/master/packages/components/Button/src/Button.settings.ts).
+
+The advantage of this approach is that the customizations are targetted to a particular component, so the changes are localized.
+
+### Changing theme tokens directly
+
+Theme tokens can be overridden directly as well. You can specify different color values for a particular set of color entries, add to the set of colors, or have different values for particular typography variants. You can see what can be overridden by looking at the [Theme type definition](https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/theme-types/src/Theme.types.ts).
+
+This approach can be preferred if you need the customizations to be applied to all components.
+
+#### Color example
+
+```ts
+color: {
+  buttonBackground: 'red',
+  neutralForeground1: theme.host.palette.Bkg,
+  otherToken: theme.colors.themePrimary,
+}
+```
+
+#### Typography example
+
+```ts
+typography: {
+  sizes: {
+    caption: 6,
+  },
+  weights: {
+    semiBold: '300',
+  },
+  families: {
+    cursive: 'Segoe UI',
+  },
+  variants: {
+    heroLargeSemibold: { face: 'cursive', size: 'heroLarge', weight: 'semiBold' }
+  }
+}
+```
+
+#### Spacing example
+
+```ts
+spacing: {
+  s1: '10px';
+}
+```
```

Comment:
Nothing actually uses spacing yet, correct? Maybe just leave it out?

Reply:
Apparently the Stack does?

Reply:
Oh? TIL...